src/content/getting-started/operations/_index.md

```diff
@@ -0,0 +1,15 @@
+---
+linkTitle: Cluster operations
+title: Cluster operations
+description: Guides to help you get going to operator your Giant Swarm clusters quickly.
```

Comment:
Not sure how to rephrase it, but your phrasing looks too complicated to me.
```suggestion
description: Guides to help you operate your Giant Swarm clusters quickly.
```

Reply:
I would agree but descriptions but me longer than 70 characters :(

Reply:
How about `Guides and best practices to help you operate your Giant Swarm clusters quickly` ?